README.md

```diff
@@ -1,44 +1,86 @@
 # App Deploy Script
 
-Deploy script used for creating tags with a tag message in the format `internal-all/v1.0.0-1234`, where `internal-all` is marking the workflow that should be started on the CI, `v1.2.3` represents app version and `1234` represents the build number. Tag message set from this script can be used later for the changelog message on any CI (e.g. Bitrise). The build number is number calculated as the total number of tags available on GitHub incremented by one.
+## Description
 
-## Installation
+Deploy script used for creating trigger tags with a tag message in the format `ci/internal-staging/2024-12-16T10-16`, where `internal-staging` is marking the workflow that should be started on the CI. The tag message set from this script can be used later for the changelog message on any CI (e.g., Bitrise). Once the build process is finished on CI/CD and the installation file is uploaded to the deployment service (e.g., TryOut Apps), an additional script is available for creating the final build tag in format `internal-staging/v1.0.0-45b46`, where `internal-staging` is marking the build type (i.e., internal for TryOut Apps, or store for App/Play Store; staging for target/flavor type), `v1.0.0-45` represents the build version (app version, build number, ...), and `b46` that represents the unique build count on CI/CD. Optionally, on some platforms, an additional value `cXX` (e.g., `c100`) can represent the code version.
```

Comment:
I would suggest a bit of a rewrite:

```suggestion
This script facilitates mobile app deployment over CI with the following features:
- Creating temporary trigger tags for starting a specific deployment workflow on the CI
- Creating final build tags by extracting environment and version data from the trigger tag and the resulting binary. 

Trigger tags use the following format: `ci/internal-staging/2024-12-16T10-16`, where `internal-staging` marks the workflow that should be started on the CI. The tag message set from this script can be used later for the changelog message on any CI (e.g., Bitrise). 
Once the build process is finished on CI/CD and the installation file is uploaded to the deployment service (e.g., TryOutApps), an additional script is available for creating the final build tag in format `internal-staging/v1.0.0-45b46`, where `internal-staging` is marking the build type (i.e., internal for TryOut Apps, or store for App/Play Store; staging for target/flavor type), `v1.0.0-45` represents the build version (app version, build number, ...), and `b46` that represents the unique build count on CI/CD. Optionally, on some platforms, an additional value `cXX` (e.g., `c100`) can represent the code version.
```

> where `internal-staging` is marking the workflow that should be started on the CI.

Maybe better "where `internal-staging` matches the trigger condition for starting the appropriate workflow on the CI"?